tests/unit/test_proxy.py

```
@@ -89,3 +162,20 @@ def test_get_value_with_value_not_existing(self):
 
         # Ensure that TestBackend's execute method was called
         self.assertIs(PyRDF.current_backend.obtained_head_node, rdf)
+
+
+class PickleTest(unittest.TestCase):
+    """Tests to check the pickling an un-pickling of Node instances."""
+
+    def test_proxy_pickle(self):
+        """
+        Test case to check that TransformationProxy objects cannot
+        be unpickled.
+        """
+
+        import pickle
+
+        node = Node(None, None)
+        proxy = TransformationProxy(node)
+        pickle.dumps(proxy)
```

As mentioned before, this method should raise an Exception, so we do not even allow users to pickle proxies.